juriscraper/opinions/united_states/state/sd.py

```diff
@@ -3,106 +3,40 @@
-        self.regex_alt = f"{regex_year_group}, (.*)"
-        self.base_path = "//table[@id='ContentPlaceHolder1_PageContent_gvOpinions']//tr[position()>1]/td"
-
-    def _get_download_urls(self):
-        path = f"{self.base_path}//a/@href[contains(.,'pdf')]"
-        return list(self.html.xpath(path))
-
-    def _get_case_names(self):
-        path = f"{self.base_path}/a[contains(@href, 'pdf')]/text()"
-        case_names = []
-        for s in self.html.xpath(path):
-            case_name = self.extract_regex_group(1, s)
-            if not case_name:
-                case_name = self.extract_regex_group(2, s, True)
-            case_names.append(titlecase(case_name.upper()))
-        return case_names
-
-    def _get_case_dates(self):
-        path = f"{self.base_path}[1]/text()"
-        return [convert_date_string(ds) for ds in self.html.xpath(path)]
-
-    def _get_precedential_statuses(self):
-        return ["Published"] * len(self.case_names)
-
-    def _get_neutral_citations(self):
-        path = f"{self.base_path}/a[contains(@href, 'pdf')]/text()"
-        neutral_cites = []
-        for s in self.html.xpath(path):
-            neutral_cite = self.extract_regex_group(2, s)
-            if not neutral_cite:
-                neutral_cite = self.extract_regex_group(1, s, True)
-            # Make the citation SD instead of S.D. The former is a neutral cite, the latter, the South Dakota Reporter
-            neutral_cites.append(neutral_cite.replace(".", "").upper())
-        return neutral_cites
 
-    def _download_backwards(self, page_year):
-        logger.info(f"Running with params: {page_year}")
-        self.initiate_webdriven_session()
-
-        # Select the year (this won't trigger a GET unless it's changed)
-        path = "//*[@id='ContentPlaceHolder1_PageContent_OpinionYears']/option[@value={year}]".format(
-            year=page_year[1]
+    def _process_html(self):
+        rows = self.html.xpath(
+            "//div[@id='ContentPlaceHolder1_ChildContent1_UpdatePanel_Opinions']//tbody/tr"
         )
-        option = self.find_element_by_xpath(path)
-        option.click()
-
-        if page_year[0] != 0:
-            # Not the first, page, go to the one desired.
-            links = self.find_elements_by_xpath(
-                "//a[@href[contains(., 'Page')]]"
+        for row in rows:
+            cite = re.findall(r"\d{4} S\.D\. \d+", get_row_column_text(row, 2))
+            if not cite:
+                continue
+            self.cases.append(
+                {
+                    "date": get_row_column_text(row, 1),
+                    "name": titlecase(
+                        get_row_column_text(row, 2).rsplit(",", 1)[0]
+                    ),
+                    "neutral_citation": cite[0],
+                    "url": row.xpath(".//td[2]/a/@href")[0],
+                    "docket": row.xpath(".//td[2]/a/@href")[0].split("/")[-1][
+                        :5
```

Comment:
Would love some examples of what the text is here.

Reply:
this may be a second candidate for extracting from the PDF.

Reply:
we extract the start of the docket.  but we dont extract the full docket.

Reply:
Sounds good. I'll hang tight for that, then.

Reply:
this has been added.